CHARTER.md

```diff
@@ -0,0 +1,104 @@
+
+*Voting Member:* A person who can elect the Governing Board and vote on changes to the MapLibre Charter.
+
+*Governing Board:* The elected representatives of the MapLibre Community who help shape the Project's overall direction and who decide over the Project's finances.
+
+*Third-party Organization:* A company or organization that uses MapLibre software for commercial or non-commercial purposes.
+
+## MapLibre Organization Goals
+The MapLibre Organization develops free open-source mapping libraries and tools for others to build applications and services.
+
+The primary goal of the MapLibre Organization is to build a friendly and innovative open-source community around the unaffiliated MapLibre Projects.
+
+The MapLibre Organization aims to encourage code development. We prefer to empower our users to build end-user applications, design maps, run map hosting services or distribute data rather than making it part of the MapLibre Projects.
+
+## Governance Bodies
+The MapLibre Organization has three Governance Bodies which support and represent the broader MapLibre Community:
+
+ * Technical Steering Committee
+ * Governing Board
+ * Voting Members
+
+### Technical Steering Committee
+The Technical Steering Committee discusses technical topics that affect multiple MapLibre projects, coordinates ongoing work, and decides if proposed changes with a major impact should be accepted or not.
+
+The committee meets on a monthly basis on a video call. 
+
+The Technical Steering Committee is open to everyone and there are no special requirements if someone wants to join it.
+
+Some projects may have their own steering committees to discuss issues relevant to just that project (e.g., native, gl-js, …).
+
+
+### Governing Board
+The Governing Board helps the Community to define our scope and direction, and is responsible for the organization finances.
+
+The Board is a group of 5 people who represent the MapLibre Organization.
+
+While the Governing Board’s purpose is not to handle technical topics in general, the Board retains the authority over all decisions the MapLibre Projects make, and may overrule the Technical Steering Committee in case of a dispute.
+
+No more than one person on the Governing Board shall be employed or contracted by a single Third-party organization.
+
+The Board shall have at least the following roles:
+
+ * Treasurer: Takes care of financial transactions and accounting.
+ * Secretary: Writes and publishes meeting notes, manages Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
```

Comment:
Is Voting membership indefinite? If so, let's keep an eye out for future motions or requirements requiring that individual votes cross some membership % threshold (because we may never be able to cross them at some point).

Reply:
That's a good catch @mojodna. It is indefinite at the moment, but we may need to limit that (or limit %-based voting to active members).

Reply:
For now we do not need to limit the duration of Voting Membership I think. What I want to make sure is that the Voting Members are informed early enough about an upcoming vote or election, that they have time to form an opinion, and that they have a very high probability of being able to take part in the vote or election. If a Voting Member then for some reason does not participate, we have to respect that.

The election or vote result will only be required to have a majority of the people who participated in the vote or election.

Reply:
I am inclined to put some notice period for votes and elections into the Charter...

Reply:
I think that resolves the issues nicely @wipfli!